tests/store/tracking/test_file_store.py

```diff
@@ -1008,6 +1008,24 @@ def test_search_runs_pagination(self):
         assert [r.info.run_id for r in result] == runs[8:]
         assert result.token is None
 
+    def test_search_runs_run_name(self):
+        fs = FileStore(self.test_root)
+        exp_id = fs.create_experiment("test_search_runs_pagination")
+        run1 = fs.create_run(exp_id, user_id="user", start_time=1000, tags=[], run_name="run_name1")
+        run2 = fs.create_run(exp_id, user_id="user", start_time=1000, tags=[], run_name="run_name2")
+        result = fs.search_runs(
+            [exp_id],
+            filter_string="attributes.run_name = 'run_name1'",
+            run_view_type=ViewType.ACTIVE_ONLY,
+        )
+        assert [r.info.run_id for r in result] == [run1.info.run_id]
+        result = fs.search_runs(
+            [exp_id],
+            filter_string="tags.`mlflow.runName` = 'run_name2'",
```

Comment:
Didn't the PR just get merged to set the tag if the `run_name` attr is set during a `create_run()` call? Why would the test fail here?

Reply:
The test was run before the PR :)

Reply:
We still need to update the search logic.

Reply:
ahhhhh gotcha!